cli/command/context/use.go

```diff
@@ -0,0 +1,25 @@
+package context
+
+import (
+	"github.com/docker/cli/cli/command"
+	"github.com/spf13/cobra"
+)
+
+func newUseCommand(dockerCli command.Cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "use CONTEXT",
+		Aliases: []string{"select", "switch"},
```

Comment:
I think these aliases are a little bit too much, why don't you just provide `--use` for this version?